Sources/LiveViewNative/Views/Layout Containers/Collection Containers/Table.swift

```diff
@@ -0,0 +1,320 @@
+                        columns[8]
+                    }
+                }
+            case 10:
+                switch selection {
+                case .single:
+                    SwiftUI.Table(rows, selection: $selection.single, sortOrder: $sortOrder) {
+                        columns[0]
+                        columns[1]
+                        columns[2]
+                        columns[3]
+                        columns[4]
+                        columns[5]
+                        columns[6]
+                        columns[7]
+                        columns[8]
+                        columns[9]
+                    }
+                case .multiple:
+                    SwiftUI.Table(rows, selection: $selection.multiple, sortOrder: $sortOrder) {
+                        columns[0]
+                        columns[1]
+                        columns[2]
+                        columns[3]
+                        columns[4]
+                        columns[5]
+                        columns[6]
+                        columns[7]
+                        columns[8]
+                        columns[9]
+                    }
+                }
+            default:
+                fatalError("Too many columns in table: \(columns.count)")
+            }
+        }
+        .applyTableStyle(element.attributeValue(for: "table-style").flatMap(TableStyle.init) ?? .automatic)
+    }
+    
+    private struct Row: Identifiable {
+        let element: ElementNode
+        var id: String {
+            guard let id = element.attributeValue(for: "id")
+            else { preconditionFailure("<table-row> must have an id") }
+            return id
+        }
+        
+        init(element: ElementNode) {
+            self.element = element
+        }
+    }
+    
+    private struct ColumnSort: SortComparator, Codable, Equatable {
+        let id: String
+        var order: SortOrder
+        
+        func compare(_ lhs: Row, _ rhs: Row) -> ComparisonResult {
+            .orderedSame
+        }
+    }
+    
+    private var columns: [TableColumn<Row, ColumnSort, some View, SwiftUI.Text>] {
+        let columnElements = element.elementChildren()
+            .filter { $0.tag == "columns" && $0.namespace == "table" }
+            .flatMap { $0.elementChildren() }
+            .filter { $0.tag == "table-column" }
+        return columnElements.enumerated().map { item in
+            TableColumn(item.element.innerText(), sortUsing: ColumnSort(id: item.element.attributeValue(for: "id") ?? String(item.offset), order: .forward)) { (row: Row) in
+                let rowChildren = row.element.children()
+                if rowChildren.indices.contains(item.offset) {
+                    context.coordinator.builder.fromNodes(
+                        [rowChildren[item.offset]],
+                        context: context
+                    )
+                    .environment(\.coordinatorEnvironment, coordinatorEnvironment)
```

Comment:
The coordinator environment values should be applied through `fromNodes`, why is it necessary to override them with the table's here?

Reply:
The `element` environment value is set in `fromNodes`, but the `coordinatorEnvironment` only seems to be set in the `NavStackEntryView`. It was crashing in `@ObservedElement` because it was missing that environment value when I scrolled down the table.

Reply:
Ah, weird that it's  not propagating through the table. Smells like a SwiftUI bug

Reply:
Probably, yeah.